terra/contracts/wormhole/src/contract.rs

```diff
@@ -261,6 +261,13 @@ fn vaa_update_guardian_set(deps: DepsMut, env: Env, data: &Vec<u8>) -> StdResult
         return ContractError::GuardianSetIndexIncreaseError.std_err();
     }
 
+    /* A new GuardianSet with zero addresses should never be accepted,
+     * it would break assumptions of the underlying contracts
+     */
+    if new_guardian_set.addresses.len() == 0
+        return ContractError::GuardianSetZeroAddresses.std_err();
```

Comment:
this error message is a little confusing, because it sounds like we have a guardian whose address is the 0 address, rather than having an empty guardian set. On that note, we should have a check that ensures we don't have a guardian with zero address (at least on EVM)

Reply:
fixed, I left the zero'd address fix out, but I think it's a good idea.  I'll create a ClickUp so I don't miss it, and make sure we get it covered for EVM as well.